client/ayon_core/pipeline/traits/generated/Ayon/traits/meta.py

```diff
@@ -94,6 +94,143 @@ def getAnnotation(self, defaultValue: str=None) -> Union[str, None]:
     
 
 
+class FilesBundleTrait:
+    """
+    A trait indicating a files bundle. Because of the limitation of the
+    current traitgen, we are using a string to represent the files,
+    sizes and hashes. This is not the final representation and will be
+    updated in the future. It is serialized as a JSON string.
+    Usage: entity
+    """
+    kId = "Ayon:meta.FilesBundle"
```

Comment:
```suggestion
    kId = "AYOn:meta.FilesBundle"
```

Reply:
Likely this requires the package bundle name to be uppercased [here](https://github.com/ynput/ayon-core/blob/b5fd14a3cc5b9b920c550a13a41b6d1aaa7e1d23/client/ayon_core/pipeline/traits/ayon_traits.yml#L3)?